File: Android/16 - TimePicker/TimePicker/app/src/main/java/com/example/timepicker/MainActivity.java
package com.example.timepicker;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.DialogFragment;

import android.os.Bundle;
import android.view.View;
import android.widget.TimePicker;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void showTimePicker(View view) {
        DialogFragment newFragment = new TimePickerFragment();
        newFragment.show(getSupportFragmentManager(),"timePicker");
    }

    public void processTimePickerResult(int hourOfDay, int minute) {
        String hour_string = Integer.toString(hourOfDay);
        String minute_string = Integer.toString(minute);
        String timeMessage = (hour_string + ":" + minute_string);

        Toast.makeText(this,getString(R.string.time_toast) + " "
                + timeMessage, Toast.LENGTH_SHORT).show();
    }
}

File: Android/Kotlin/ExercicioLinearLayoutcheckboxeimageview/app/src/main/java/com/example/exerciciolinearlayoutcheckboxeimageview/MainActivity.kt
package com.example.exerciciolinearlayoutcheckboxeimageview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun isChecked(view: View) {
        if(checkBox.isChecked){
            imageView.setColorFilter(resources.getColor(R.color.vermelho))
        } else{
            imageView.setColorFilter(resources.getColor(R.color.black))
        }
    }
}
